tests/unittest/disaggregated/test_kv_transfer.py

```
@@ -1047,5 +1048,265 @@ def test_transfer_worker_v2_with_window(
+            tokens_per_block=setup["tokens_per_block"],
+        )
+        kv_slice = KVSlice(is_last_slice=True, block_ids_per_layer_groups=block_ids_per_groups)
+        future = tx_session.send(kv_slice)
+
+        # No receiver registered yet; task is INIT.
+        tx_session.cancel()
+
+        assert tx_session.status == SessionStatus.CANCELLED
+        assert tx_session.has_failed()
+        # Future for the cancelled INIT task must raise.
+        with pytest.raises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)